backend/src/test/java/com/crm/service/impl/CarServiceImplTest.java

```diff
@@ -60,29 +60,29 @@ public static void setUp() {
     }
 
     @Test
-    public void getCarByIdShouldReturnCorrectResponse() {
+    public void shouldGetCarById() {
         final Optional<CarEntity> optionalCarEntity = Optional.of(CarServiceImplTest.carEntity);
-        when(carRepository.findById(ONE)).thenReturn(optionalCarEntity);
+        when(carRepository.findById(ID)).thenReturn(optionalCarEntity);
         when(carMapper.convertToDto(carEntity)).thenReturn(carResponse);
 
-        carService.getCarById(ONE);
+        carService.getCarById(ID);
 
-        verify(carRepository, times(1)).findById(ONE);
+        verify(carRepository, times(1)).findById(ID);
         verify(carMapper, times(1)).convertToDto(carEntity);
     }
 
-    @Test(expected = CarNotFoundException.class)
-    public void getCarByIdShouldThrowExceptionForNotExistingEntry() {
-        when(carRepository.findById(ONE)).thenReturn(Optional.empty());
+    @Test(expected = ResourceNotFoundException.class)
```

Comment:
also consider change of exception like above in my comments

Reply:
Done